### PR TITLE
Fixup description mapping and formatting/style issues

### PR DESCRIPTION
**Pull request by moltude**
Maps `description` rather than `description_ss` and cleaned up formatting with rubocop.

### CODE REVIEW

File: heidrun/cdl.rb [hunk 1]
```diff
@@ -12,9 +12,7 @@
   md5_hash = r['reference_image_md5'].first ? r['reference_image_md5'].first.value : nil
   image_url = nil
 
-  if !md5_hash.nil?
-    image_url = base_url + '150x150/' + md5_hash
-  end
+  image_url = base_url + '150x150/' + md5_hash unless md5_hash.nil?
```
Comment (no-reply) [inline heidrun/cdl.rb]: Could this whole block all be flattened:

``` ruby
md5_hash = r['reference_image_md5'].first && r['reference_image_md5'].first.value
md5_hash && ("https://thumbnails.calisphere.org/clip/150x150/#{md5_hash}")
```

I'm okay with expanding this out for readability as desired, but the current version feels overly complex to me.

Comment (moltude) [inline heidrun/cdl.rb]: Yours works for me.

**Comment by moltude** [comment]
I'm not sure if this should be :squash or not but I think it is ready

**Comment by moltude** [comment]
@no-reply rewrote #cdl_preview after #kdl_preview and tested locally.

**Comment by no-reply** [comment]
Looks good. I'm :+1: to merge; pending your local test.

**Pull request merged**